src/components/Header/components/PeopleSearch/index.js

```diff
@@ -138,7 +149,7 @@ export default class GordonPeopleSearch extends Component {
     var hasMatched = false;
     return (
       <span>
-        {parts.map((part) =>
+        {parts.map(part =>
```

Comment:
Do the majority of these `<someParam> => ...` style actions have parentheses around the `<someParam>`? I think I just saw a pull request the other day from one of the 360 team that added the parentheses to each instance of such syntax.

Reply:
This was done by the Prettier formatter, it likes to do that a lot, but also sometimes Prettier will specifically remove them too. As a guess: maybe it adds them when it is in a JSX expression inside the `{ ... }` but maybe it removed them if it is a method body?
TLDR: Idk why, but Prettier adds parentheses in some cases and removes them in other cases

Reply:
Huh, okay. It's fine then.